### PR TITLE
"Enabled Reassigning of Contact Workflow"

### PR DESCRIPTION
**Pull request by cosmicmeditator**

### CODE REVIEW

File: src/main/java/com/kylas/sales/workflow/domain/workflow/action/ValueConverter.java [hunk 1]
```diff
@@ -59,9 +61,17 @@ public Object getValue(EditPropertyAction editPropertyAction, Field field, Entit
...
+            return editPropertyAction.getName().equals(PIPELINE.getName())
+                ? convertToObject(objectMapper.readValue(valueOf(editPropertyAction.getValue()), IdName.class).getId(), field)
+                : convertToObject(editPropertyAction.getValue(), field);
+          }
+          if (entityType.equals(CONTACT)) {
+            return editPropertyAction.getName().equals(COMPANY.getName())
+                ? convertToObject(objectMapper.readValue(valueOf(editPropertyAction.getValue()), IdName.class).getId(), field)
+                : convertToObject(editPropertyAction.getValue(), field);
+          }
+          return convertToObject(editPropertyAction.getValue(), field);
```
Comment (bhaskarpawar123) [inline src/main/java/com/kylas/sales/workflow/domain/workflow/action/ValueConverter.java:74]: We have to refactor this please do remember next time..

**Review by bhaskarpawar123 — APPROVED** [review]

File: src/test/java/com/kylas/sales/workflow/config/TestDatabaseInitializer.java [hunk 2]
```diff
@@ -6,11 +6,13 @@
 import org.springframework.context.ApplicationContextInitializer;
 import org.springframework.context.ConfigurableApplicationContext;
 import org.springframework.data.jpa.repository.config.EnableJpaRepositories;
+import org.springframework.transaction.annotation.EnableTransactionManagement;
 import org.testcontainers.containers.PostgreSQLContainer;
 import org.testcontainers.junit.jupiter.Container;
 
 @TestConfiguration
 @EnableJpaRepositories
+@EnableTransactionManagement
```
Comment (abhishek-bhuskute) [inline src/test/java/com/kylas/sales/workflow/config/TestDatabaseInitializer.java:15]: Do we need this? Shouldn't this be done by boot on its own?

**Review by abhishek-bhuskute — CHANGES_REQUESTED** [review]
Just check if you need `@EnableTransactionManagement`.

File: src/main/java/com/kylas/sales/workflow/domain/workflow/action/ValueConverter.java (hunk 1 above)
Comment (abhishek-bhuskute) [reply src/main/java/com/kylas/sales/workflow/domain/workflow/action/ValueConverter.java:74]: Let's do it this time only.

**Review by abhishek-bhuskute — APPROVED** [review]

**Review by bhaskarpawar123 — APPROVED** [review]

**Pull request merged**